components/kserve/kserve_config_handler.go

```diff
@@ -25,18 +25,22 @@ const (
 func (k *Kserve) setupKserveConfig(ctx context.Context, cli client.Client, logger logr.Logger, dscispec *dsciv1.DSCInitializationSpec) error {
 	// as long as Kserve.Serving is not 'Removed', we will setup the dependencies
 
+	var defaultDeploymentMode DefaultDeploymentMode
+	var disableIngressCreation bool
+	if k.DefaultDeploymentMode == "" {
+		defaultDeploymentMode = Serverless
+	} else {
+		defaultDeploymentMode = k.DefaultDeploymentMode
+	}
+	if !k.DisableIngressCreation {
+		disableIngressCreation = true
+	} else {
+		disableIngressCreation = k.DisableIngressCreation
+	}
```

Comment:
this condition means `disableIngressCreation = true`

Reply:
fixed